packages/react-composites/tests/browser/call/hermetic/HoldScreen.test.ts

```diff
@@ -3,13 +3,12 @@
 
 import { expect } from '@playwright/test';
 import { IDS } from '../../common/constants';
-import { dataUiId, isTestProfileStableFlavor, pageClick, stableScreenshot, waitForSelector } from '../../common/utils';
+import { dataUiId, pageClick, stableScreenshot, waitForSelector } from '../../common/utils';
 import { buildUrlWithMockAdapter, defaultMockCallAdapterState, defaultMockRemoteParticipant, test } from './fixture';
 
+/* @conditional-compile-remove(PSTN-calls) @conditional-compile-remove(one-to-n-calling) */
```

Comment:
I thought when denoting 2 features in a CC statement it was something like `@conditional-compile-remove(feature 1 feature 2)` when in the same comment?

Reply:
Nah, we don't support fancy expressions inside the directive. But we do support multiple directives in the same / different comment.

See https://github.com/Azure/communication-ui-library/blob/b7666daf66fdb29053a89ee516a839ece5a7d9b1/packages/acs-ui-common/src/conditional-compilation-sample/index.tsx#L233